src/Sulu/Bundle/MediaBundle/Search/Subscriber/StructureMediaSearchSubscriber.php

```diff
@@ -118,14 +118,13 @@ private function getImageUrl($data, $locale)
             $medias = $data->getData('de');
         // old ones an array ...
         } else {
-            if (!isset($data['ids'])) {
-                throw new \RuntimeException(
-                    \sprintf('Was expecting media value to contain array key "ids", got: "%s"', \print_r($data, true))
-                );
+            $ids = $data['ids'] ?? $data['id'] ?? null;
```

Comment:
Will `$data['id']` not return an int and so !is_array will always fail?

Reply:
Just realized that as well :sweat_smile: Already working on it